cmd/finch/nerdctl_test.go

```diff
@@ -365,144 +365,6 @@ func TestNerdctlCommand_run(t *testing.T) {
 				logger.EXPECT().Debugf("Status of virtual machine: %s", "Running")
 			},
 		},
-		{
```

Comment:
should we still keep these tests or is upstream testing in nerdctl sufficient ? cc: @ningziwen

Reply:
These are unit tests so I think they are not related to upstream testing. The test parity of nerdctl upstream tests here should be Finch e2e tests instead of unit tests. We already have them covered.

From what I understand, the unit tests should strictly test the unit. Now the special handling logic is removed from nerdctl.go so the unit tests are also being removed from nerdctl_tests.go. We already mocked nerdctl in unit tests so any logic inside nerdctl should not be tested here.